content/departments/cloud/index.md

```diff
@@ -195,6 +195,14 @@ We maintain an [on-call rotation in Opsgenie](https://sourcegraph.app.opsgenie.c
 - Initiating incident procedures
 - Publishing postmortems
 
+**Expectations of work while on-call**
+Oncall engineers are explicitly empowered to work on reliability issues and other high priority work while on-call. This is a critical part of the role and is expected to be a significant portion of the on-call engineer's time. The on-call engineer is expected to be available to respond to alerts and to initiate incident procedures.
```

Comment:
This doesn't seem to render the way we want:

<img width="408" alt="CleanShot 2022-10-24 at 11 15 34@2x" src="https://user-images.githubusercontent.com/2946214/197441771-eab40f24-598f-4fca-bbb2-084a29b68ef2.png">

```suggestion
#### Expectations of work while on-call

Oncall engineers are explicitly empowered to work on reliability issues and other high priority work while on-call. This is a critical part of the role and is expected to be a significant portion of the on-call engineer's time. The on-call engineer is expected to be available to respond to alerts and to initiate incident procedures.
```